<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ControlWindow</class>
 <widget class="QMainWindow" name="ControlWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>990</width>
    <height>616</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>BotFather - Control Window</string>
  </property>
  <property name="dockOptions">
   <set>QMainWindow::AllowTabbedDocks|QMainWindow::AnimatedDocks</set>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="BrowserWidget" name="label">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Hit the â–· button to run a bot script.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>990</width>
     <height>25</height>
    </rect>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::PreventContextMenu</enum>
   </property>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionScripts"/>
    <addaction name="actionSupport"/>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::PreventContextMenu</enum>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionStart"/>
   <addaction name="actionStop"/>
   <addaction name="separator"/>
   <addaction name="actionLog"/>
   <addaction name="actionSettings"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionStart">
   <property name="icon">
    <iconset resource="../res/gui.qrc">
     <normaloff>:/gui/win10-Play-96.png</normaloff>:/gui/win10-Play-96.png</iconset>
   </property>
   <property name="text">
    <string>Start</string>
   </property>
   <property name="toolTip">
    <string>Start the bot</string>
   </property>
  </action>
  <action name="actionStop">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="../res/gui.qrc">
     <normaloff>:/gui/win10-Stop-96.png</normaloff>:/gui/win10-Stop-96.png</iconset>
   </property>
   <property name="text">
    <string>Stop</string>
   </property>
   <property name="toolTip">
    <string>Stop the bot</string>
   </property>
  </action>
  <action name="actionSettings">
   <property name="icon">
    <iconset resource="../res/gui.qrc">
     <normaloff>:/gui/win10-SettingsMixer-96.png</normaloff>:/gui/win10-SettingsMixer-96.png</iconset>
   </property>
   <property name="text">
    <string>Settings</string>
   </property>
   <property name="toolTip">
    <string>Open the settings dialog</string>
   </property>
  </action>
  <action name="actionSupport">
   <property name="text">
    <string>Get Support</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About Botfather</string>
   </property>
  </action>
  <action name="actionScripts">
   <property name="text">
    <string>Get Scripts</string>
   </property>
  </action>
  <action name="actionLog">
   <property name="icon">
    <iconset resource="../res/gui.qrc">
     <normaloff>:/gui/win10-Ingredients List-96.png</normaloff>:/gui/win10-Ingredients List-96.png</iconset>
   </property>
   <property name="text">
    <string>Log</string>
   </property>
   <property name="toolTip">
    <string>Open the log window</string>
   </property>
  </action>
  <action name="actionKill">
   <property name="icon">
    <iconset resource="../res/gui.qrc">
     <normaloff>:/gui/win10-delete-96.png</normaloff>:/gui/win10-delete-96.png</iconset>
   </property>
   <property name="text">
    <string>Kill</string>
   </property>
   <property name="toolTip">
    <string>Kill the script</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>BrowserWidget</class>
   <extends>QLabel</extends>
   <header location="global">gui/browser_widget.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../res/gui.qrc"/>
 </resources>
 <connections/>
</ui>
